apps/vaporgui/MainForm.cpp

```diff
@@ -1929,7 +1929,12 @@ void MainForm::_performSessionAutoSave()
 
     if (_eventsSinceLastSave >= eventCountForAutoSave) {
         string autoSaveFile = sParams->GetAutoSaveSessionFile();
-        _paramsMgr->SaveToFile(autoSaveFile);
+        int    rc = _paramsMgr->SaveToFile(autoSaveFile);
+        if (rc == -1) {
+            MSG_ERR("Unable to open settings file " + autoSaveFile);
```

Comment:
Wouldn’t this be the session file rather than the settings file? Also we may want to make this a warning that can be disabled rather than an error.

Reply:
You're correct.  I removed all directory references (not just the settings) that used "~" because ofstream will not create a new file without a full file path.